start.sh

```diff
@@ -47,7 +47,7 @@ else
   if ( $fast ); then
     sed -i 's/development/production/' .env
   fi
-  vagrant box list | grep "202303.13.0" -q || vagrant box add bento/ubuntu-18.04 --provider virtualbox --box-version "202303.13.0" -c
+  vagrant box list | grep "202303.13.0" -q || vagrant box add bento/ubuntu-22.04 --provider virtualbox --box-version "202303.13.0" -c
   vagrant plugin list | grep vbguest -q || vagrant plugin install vagrant-vbguest
```

Comment:
Can you add `vagrant plugin install vagrant-vbguest` to the installation instructions for non-linux? Since they do not use start.sh but go to use `vagrant up` immediately. I just noticed it when booting things up from scratch on my Windows laptop